pkg/internal/testhelpers/containers.go

```diff
@@ -158,20 +385,25 @@ func StartPGContainerWithImage(ctx context.Context, image string, testDataDir st
 		Env: map[string]string{
 			"POSTGRES_PASSWORD": "password",
 		},
-		SkipReaper: false, /* switch to true not to kill docker container */
+		Networks:       networks,
+		NetworkAliases: map[string][]string{"promscale-network": {"db" + containerPort.Port()}},
+		SkipReaper:     false, /* switch to true not to kill docker container */
 	}
 	req.Cmd = []string{
 		"-c", "max_connections=100",
+		"-c", "port=" + containerPort.Port(),
+		"-c", "max_prepared_transactions=150",
+		"-i",
```

Comment:
what does the -i do?

Reply:
> Allows remote clients to connect via TCP/IP (Internet domain) connections. Without this option, only local connections are accepted. This option is equivalent to setting listen_addresses to * in postgresql.conf or via -h.

I figured it pays to be explicit about this, in case we decide to tighten up the `postgresql.conf`